src/Catalyst.Cli.Tests/UnitTests/Commands/Response/GetMempoolResponseTests.cs

```diff
@@ -42,7 +42,7 @@ public void GetMempoolResponse_Can_Get_Output()
             //Arrange
             var getMempoolResponse = new GetMempoolResponse();
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var getMempoolCommand = new GetMempoolCommand(commandContext, Substitute.For<ILogger>());
+            new GetMempoolCommand(commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'GetMempoolCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)